install.md

```diff
@@ -83,8 +83,10 @@ xUbuntu 18.04
 To install, choose a supported version for your operating system, and export it
 as a variable, like so: `export VERSION=1.19`
 
-We also save releases as subprojects. If you'd, for instance, like to use `1.19.1`
-you can set `export VERSION=1.19:1.19.1`
+We also save releases as subprojects. If you'd, for instance, like to use `1.24.5`
+you can set
+`export VERSION=1.24.5` and
```

Comment:
sorry two more nits:

can this line be compressed to 1
can your commits all be squashed together?

Reply:
The last commit contains all changes. I can't remove the previous ones, can I?

Reply:
you can use `git rebase -i upstream/main` where upstream is the remote used for the upstream repo like `git@github.com:cri-o/cri-o.git` and then select `f` instead of pick for all but the first commit to squash all of the commits together